app/assets/stylesheets/utilities/_typography.scss

```diff
@@ -16,6 +16,7 @@ body { -webkit-font-smoothing: antialiased; }
 .fs-12p { font-size: 12px; }
 .fs-13p { font-size: 13px; }
 .fs-20p { font-size: 20px; }
+.fs-24p { font-size: 24px; }
```

Comment:
Personally I'd like to see us move away from these ad-hoc font size classes toward the standard set of [design system utility classes](https://designsystem.digital.gov/utilities/font-size-and-family/#utility-font). There's not a great fit for precisely 24px, at least within the monospace family of text utility (choice between 21px and 30.5px). We could misuse `font-body-xl` for just the font-size to get pretty close at 23.4px.

That being said, per my previous comment, if we just set the font-size on `.separator-text {}` styles, we could probably avoid the utility class altogether.

Reply:
I thought about using `font-ui-large`, but I did not because there was not an exact match at 24px. Since we are trying to opt for system utility classes, I will use this and make sure it passes visual testing.

Reply:
> I thought about using `font-ui-large`, but I did not because there was not an exact match at 24px. Since we are trying to opt for system utility classes, I will use this and make sure it passes visual testing.

Yeah, alternatively, if we want 24px exactly, we could set it for the `separator-text` styles, and remove the font class altogether on the `<code>` wrappers?

Like in the example above:

```scss
.separator-text {
  // ...
  font-size: 1.5rem;
}
```